### ISSUE TITLE
NER stop list not working

### ISSUE THREAD

**Issue by enoriega** [clulab/reach#227]
The NER stop list in bioresources is not being followed. I added a unit test that checks whether ReachSystem does't extract BioMentions from the individual elements in the list.

I suspect this will be fixed once Reach is updated to use the latest version of processors

**Comment by hickst** [comment]
Please test this theory by simply publishing local Processors, then changing reach build to use 5.8.5-SNAPSHOT and re-testing. This should take you 5 minutes to setup for running Reach tests.

**Comment by enoriega** [comment]
I did already and it is true

**Comment by hickst** [comment]
great, thanks

**Comment by MihaiSurdeanu** [comment]
Close the issue?

**Comment by enoriega** [comment]
Please don't, the test doesn't pass yet in the committed code.

**Comment by hickst** [comment]
I'm testing this with Bioresources 1.1.9 (commit 9f9feb4) and Processors 5.8.5-SNAPSHOT (commit 0c84df) and it's not working correctly with initial-cap stop words. Here are some examples at a Reach shell which is using the above versions:
```>>> role
sentence #0
role
Tokens: (0,role,NN)
role
roots: 0
outgoing:
incoming:


entities:

events:
==================================================
>>> Role
sentence #0
Role
Tokens: (0,Role,NN)
role
roots: 0
outgoing:
incoming:


entities:
mention text: Role
List(Organ, Context)
	------------------------------
	Rule => ner-organ
	Type => CorefTextBoundMention
	------------------------------
	Organ|List(Organ, Context) => Role
	grounding: KBResolution(role, uberon, UBERON:2002175, )
	------------------------------
```
and 
```
>>> cells
sentence #0
cells
Tokens: (0,cells,NNS)
cell
roots: 0
outgoing:
incoming:


entities:

events:
==================================================
>>> Cells
sentence #0
Cells
Tokens: (0,Cells,NNS)
cell
roots: 0
outgoing:
incoming:


entities:
mention text: Cells
List(CellType, Context)
	------------------------------
	Rule => ner-cell-type
	Type => CorefTextBoundMention
	------------------------------
	CellType|List(CellType, Context) => Cells
	grounding: KBResolution(cells, cl, CL:0000000, )
	------------------------------
```

**Comment by MihaiSurdeanu** [comment]
Fixed in the processors 5.8.5-SNAPSHOT master branch.
